turbinia/config/recipes/triage-linux.yaml

```diff
@@ -0,0 +1,23 @@
+# Triage recipe for quick initial response (specific to Linux).
+
+globals:
+  jobs_allowlist:
+    - CronExtractionJob
+    - CronAnalysisJob
+    - PlasoJob
+    - PsortJob
+    - FileSystemTimelineJob
+
+plaso_base:
+  task: 'PlasoTask'
+
+  file_filter: [
+      '/var/log/\w+[.]log([.]\d.gz|[.]\d|$)',
+      '/var/log/audit/audit[.]log([.]\d[.]gz|[.]\d|$)',
```

Comment:
I wonder if we want to add the cron files to the timeline as well?

Reply:
Done